tests/visualization_tests/matplotlib_tests/test_parallel_coordinate.py

```diff
@@ -19,40 +23,98 @@ def test_target_is_none_and_study_is_multi_obj() -> None:
         plot_parallel_coordinate(study)
 
 
+def _fetch_objectives_from_figure(figure: Axes) -> List[float]:
+    # Fetch line plots in parallel coordinate.
+    line_collections = figure.findobj(LineCollection)
+    assert len(line_collections) == 1
+
+    # Fetch objective values from line plots.
+    objectives = [line[0, 1] for line in line_collections[0].get_segments()]
+    return objectives
+
+
+def _test_xtick_labels(axes: Axes, expected_labels: List[str]) -> None:
+    xtick_labels = axes[0].get_xticklabels()
+    for expected_label, xtick_label in zip(expected_labels, xtick_labels):
```

Comment:
How about checking the length of `expected_labels` and `xtick_labels` are same? Using the `zip`, tests will end up seeing only the smaller length one when the lengths are different.

Reply:
Absolutely. Indeed, a few `expected_labels` contain unnecessary elements, which should be fixed.